static/app/components/dropdownMenu.tsx

```diff
@@ -216,6 +216,8 @@ export default DropdownMenu;
 
 const StyledOverlay = styled(Overlay)`
   max-width: 24rem;
+  overflow-y: scroll;
+  max-height: 90vh;
```

Comment:
Will this still apply if the dropdown is places somewhere in the middle or bottom of the page?

Reply:
yes... but it falls apart pretty quickly lol. admittedly i didn't spend much time on this, and its quite naive.

`max-height` should be more of a sane default rather than `90vh`. the `popper` implementation would also need to change to allow for overflow as currently if you near an edge it still auto closes.

i figured my quick fix would cover the vast majority of scenarios.

fwiw, we also have other select fields that look the same and can scroll... we could easily have used one of these. not sure why we have such a variety in this department.

Reply:
that's what's I'm implementing right now: a `popper` modifier that automatically restricts the overlay size to prevent overflowing. it's working but I still need to deal with some edge cases:
https://user-images.githubusercontent.com/44172267/202538628-6b99be58-3add-4f8c-98ad-809bb98eaaa6.mov